dev_tools/docs/nxdl.py

```diff
@@ -512,10 +512,16 @@ def _print_enumeration(self, indent, ns, parent):
         if len(node_list) == 0:
             return ""
 
-        if len(node_list) == 1:
-            self._print(f"{indent}Obligatory value:", end="")
+        if parent.attrib.get("open", "false") == "true":
+            self._print(
+                f"{indent}This is an open enumeration. Any of these values is suggested:",
```

Comment:
The phrase open enumeration is good for us developers but I'm not sure how easy it is to understand for non-developers.  How about:

```suggestion
                f"{indent}Any of these values or a custom value",
```